### PR TITLE
WebDriver: Fix POST /session/:session_id/elements to return multiple

### PR DESCRIPTION
**Pull request by JRaspass**
Very unfamiliar with rust, but I'm trying to test servo with my Perl WebDriver module ( https://metacpan.org/pod/WebDriver::Tiny ) and fix any issues I come across. I hope to make my next commit implement find with the location strategy of link text.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [X] These changes do not require tests because I don't believe there are WebDriver tests, I'd love to be wrong and pointed in the right direction though :-P

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/12218)

### CODE REVIEW

**Comment by jdm** [comment]
[Webdriver tests](tests/wpt/web-platform-tests/webdriver) are quite new, and we don't support them yet. Thanks for the fix!

@bors-servo: r+

File: components/webdriver_server/lib.rs [hunk 1]
```diff
@@ -561,7 +561,7 @@ impl Handler {
         }
 
         let (sender, receiver) = ipc::channel().unwrap();
-        try!(self.frame_script_command(WebDriverScriptCommand::FindElementCSS(parameters.value.clone(),
+        try!(self.frame_script_command(WebDriverScriptCommand::FindElementsCSS(parameters.value.clone(),
                                                                               sender)));
```
Comment (Ms2ger) [inline components/webdriver_server/lib.rs]: Please fix the indentation here; should be one more space.

**Pull request merged**